Interview Questions

```diff
@@ -0,0 +1,78 @@
+# Interview questions for Junior Front-End Web Developers
+
+The purpose of this article is to familiarize fresh Front-End Web Developers with text book questions. If you are just getting into the developer world there is a good chance you will feel nervous when interviewing for a developer's positon. Because of this, some relatively basic questions could feel more difficult then how they actually are. A good counter for this scenario is to be properly prepared. Without further ado, here are a few questions that you could encounter:
+
+1)	Can you explain the meaning of “inline-block”?  
+
+
+
+2)	If you have and ul element with several li elements, how would you go about setting up these elements in a row by using HTML and CSS? 
+
+
+3)	What is the meaning of floats? How do they work? When would you use floats?
+
+
+4)	When would you use:  clear: both;?
+
+
+5)	What is Bootstrap, how does it work and why did you use it?
+
+
+6)	Explain the notion of responsive design and how it works? Use an example if you like.
+
+
+7)	What is DOM? And how would you explain it with your own words. 
+
+
+8)	What is ECMA? 
+
+
+9)	What is jQuery and what is it commonly used for? 
+
+
+10)	What is the difference between == and === in JS? 
+
+
+11)	What JavaScript types have you used? 
+
+
+12)	What is a callback function in JS and when would you use one? 
+
+
+13)	What is the difference between undefined and NULL? 
+
+
+14)	What is an async response? 
+
+
+15)	What is the meaning of JSON? When is it used?
```

Comment:
I don't think 
> when is it used?

is a good way to phrase this question.

JSON is fat free XML. And the places where JSON is used, you cannot use anything else. JSON is part of the protocol. If the server sends you JSON, you have to parse the JSON. 

Or use a different URI that sends XML or maybe uses protocol buffers.